lib/centralisation/ui/pages/Main.dart

```diff
@@ -65,82 +95,104 @@ class LinksScreen extends HookConsumerWidget {
-                            title: Row(
-                              mainAxisAlignment: MainAxisAlignment.spaceBetween,
-                              children: [
-                                Expanded(
-                                  child: Align(
-                                    alignment: Alignment.centerLeft,
-                                    child: Text(e.name),
-                                  ),
+                  onLongPressStart: (_) {
+                    _handleLongPressStart(context, e);
+                  },
+                  onLongPressEnd: (_) {
+                    _handleLongPressEnd();
+                  },
+                  child: Container(
+                    margin: EdgeInsets.only(bottom: 10.0),
+                    decoration: BoxDecoration(
+                      border: Border.all(color: Colors.black12, width: 3),
+                      borderRadius: BorderRadius.circular(8.0),
+                    ),
+                      child: Container(
+                        decoration: BoxDecoration(
+                          color: Colors.white,
+                          borderRadius: BorderRadius.circular(8.0),
+                        ),
+                        width: MediaQuery.of(context).size.width / 1  - 0.05 * MediaQuery.of(context).size.width,
+                        height: 70,
+                        child: ListTile(
+                          contentPadding: EdgeInsets.zero,
+                          title: Row(
+                            children: [
+                              e.icon.toLowerCase().endsWith('.svg')
+                                  ? Container(
+                                margin: EdgeInsets.only(left: 10.0,right: 10.0, bottom : 3),
+                                child: SvgPicture.network(
+                                  "https://centralisation.eclair.ec-lyon.fr/assets/icons/" + e.icon,
                                 ),
-                                IconButton(
-                                  icon: favorites.contains(e)
-                                      ? Icon(Icons.star)
-                                      : Icon(Icons.star_border),
-                                  onPressed: () {
-                                    toggleFavorite(e);
-                                  },
+                                width: 45,
+                                height: 45,
+                              )
+                                  : Container(
+                                margin: EdgeInsets.only(left: 10.0,right: 10.0, bottom : 3),
+                                child: Image.network(
+                                  "https://centralisation.eclair.ec-lyon.fr/assets/icons/" + e.icon,
+                                ),
+                                width: 45,
+                                height: 45,
+                              ),
+                              SizedBox(width: 10), // Ajoute un espace entre le logo et le texte
```

Comment:
I think that you can delete comment, unless it makes it easier for you to understand